en/more-08.md

```diff
@@ -2,6 +2,194 @@
+  \multicolumn{1}{l}{Animal} & Food  & Size   \\
+  \midrule
+  dog    & meat  & medium \\
+  horse  & hay   & large  \\
+  frog   & flies & small  \\
+  \bottomrule
+\end{tabular}
+\end{document}
+```
+<!-- {% endraw %} -->
+
+### Manipulating the space between columns
+
+Usually LaTeX pads each column by some space on both ends to give a balanced
+look and separate them. This space is defined with the length `\tabcolsep`. Due
+to the fact that each column is padded on both sides you get one `\tabcolsep` on
+either end of the table, and `2\tabcolsep` between two columns &ndash; one from
+each column. You can adjust this space to any length using `\setlength`:
+
+<!-- {% raw %} -->
+```latex
+\documentclass{article}
+\usepackage{array}
+
+\setlength\tabcolsep{1cm}
+
+\begin{document}
+\begin{tabular}{lll}
+  Animal & Food  & Size   \\
+  dog    & meat  & medium \\
+  horse  & hay   & large  \\
+  frog   & flies & small  \\
+\end{tabular}
+\end{document}
+```
+<!-- {% endraw %} -->
+
+You can change this space to something arbitrary using `@`. This will remove the
+padding between two columns or on either end, and instead put anything inbetween
+the columns you specify as an argument:
+
+<!-- {% raw %} -->
+```latex
+\documentclass{article}
+\usepackage{array}
+
+\begin{document}
+\begin{tabular}{l@{ : }l@{\hspace{2cm}}l}
+  Animal & Food  & Size   \\
+  dog    & meat  & medium \\
+  horse  & hay   & large  \\
+  frog   & flies & small  \\
+\end{tabular}
+\end{document}
+```
+<!-- {% endraw %} -->
+
+(We'll see `\hspace` [again shortly](lesson-11); you might guess that it adds a
+horizontal space.)
+
+The `!` preamble token does something pretty similar. The difference is, that it
+_adds_ its argument to the space in the centre between two columns.
+
+<!-- {% raw %} -->
+```latex
+\documentclass{article}
+\usepackage{array}
+
+\begin{document}
+\begin{tabular}{l!{:}ll}
+  Animal & Food  & Size   \\
+  dog    & meat  & medium \\
+  horse  & hay   & large  \\
+  frog   & flies & small  \\
+\end{tabular}
+\end{document}
+```
+<!-- {% endraw %} -->
+
+
+### Vertical rules
+
+Though you _don't_ want to use vertical rules, sometimes you just have to. LaTeX
+still got you covered.
+
+<!-- {% raw %} -->
+```latex
+\documentclass{article}
+\usepackage{array}
+
+\begin{document}
+\begin{tabular}{l|ll}
+  Animal & Food  & Size   \\
+  dog    & meat  & medium \\
+  horse  & hay   & large  \\
+  frog   & flies & small  \\
+\end{tabular}
+\end{document}
+```
+<!-- {% endraw %} -->
+
+You might notice that the behaviour of `|` is pretty similar to `!{decl}`, it
+adds the vertical rule between two columns leaving the padding as it is. There
+is a huge downside of this though, vertical rules don't work with the pretty
+horizontal rules provided by `booktabs`. You can use the horizontal rules
+provided by LaTeX, those are `\hline` (equaling `\toprule`, `\midrule`, and
+`\bottomrule`) and `\cline` (which behaves like `\cmidrule`).
+
+## Customising `booktabs` rules
```

Comment:
`Customising` → `Customizing` (besides the paragraph below also uses the US English spelling)